zerver/webhooks/clubhouse/doc.md

```diff
@@ -2,13 +2,15 @@ Get Zulip notifications for your Clubhouse Stories and Epics!
 
 1. {!create-stream.md!}
 
-1. {!create-bot-construct-url.md!}
+1. {!create-an-incoming-webhook.md!}
+
+1. [Generate the URL]() for your bot.
```

Comment:
I think "generate the URL for your bot" isn't quite right -- this URL is for a particular incoming webhook setup for a bot. For some integrations, it will be common to generate 10 separate URLs and configure them with 10 separate apps. (Say, GitHub Bot being setup with each of 10 different repositories, each going to a different stream).

Reply:
So "for your intergration"?